reproductions/Log4Net.SerializationException/Log4Net.SerializationException.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net452;net461</TargetFrameworks>
+    <IsPackable>false</IsPackable>
+    <ExcludeManagedProfiler>true</ExcludeManagedProfiler>
+    <ExcludeNativeProfiler>true</ExcludeNativeProfiler>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\reproduction-dependencies\ApplicationWithLog4Net\ApplicationWithLog4Net.csproj" />
+  </ItemGroup>
+
+  <Target Name="AfterBuildMoveApplicationWithLog4Net" AfterTargets="AfterBuild">
```

Comment:
One outstanding item is automating this project in CI. Once that's done, this should be complete.

Reply:
Completed in d7575ea82b9b5c0993892c6e240f902ddfe5a384

Reply:
👍